app/models/listing.rb

```diff
@@ -0,0 +1,10 @@
+class Listing < ApplicationRecord
+  belongs_to :user
+  has_many_attached :images
+
+  validates :title, :authors, :description, :listing_type, :university, presence: true
```

Comment:
Let's update the migration for the listings to make sure these fields aren't null at the database level.